src/assets/css/footer-section.css

```diff
@@ -0,0 +1,27 @@
+.media-icon{
```

Comment:
- Follow `2 space` indentation for `.css` files.
- I think we can move some custom styles to tailwind config file (@roseaugusto?)